src/hyperion/external_interaction/callbacks/rotation/nexus_callback.py

```diff
@@ -95,5 +95,8 @@ def activity_gated_start(self, doc: RunStart):
                 shape,
                 parameters.scan_points,
                 omega_start_deg=parameters.omega_start_deg,
-                chi_start_deg=parameters.chi_start_deg or 0,
+                chi_phi_start_deg=(
```

Comment:
Could: I would rather either these all be separate params, or better yet an `OptionalGonioAngleStarts` so that the defaults only have to be defined in one place